### PR TITLE
fix(ios): accidental duplicated line from merge

### PR DESCRIPTION
**Pull request by jahorton**

### CODE REVIEW

File: ios/engine/KMEI/KeymanEngine/Classes/Resource Management/PackageInstallViewController.swift [hunk 1]
```diff
@@ -349,8 +349,6 @@ public class PackageInstallViewController<Resource: LanguageResource>: UIViewCon
     Manager.shared.shouldReloadKeyboard = true
     self.pickingCompletionHandler(selectedResources.map { $0.typedFullID })
 
-    self.completionHandler(selectedResources.map { $0.typedFullID })
```
Comment (jahorton) [inline ios/engine/KMEI/KeymanEngine/Classes/Resource Management/PackageInstallViewController.swift:352]: `completionHandler` was renamed to `pickingCompletionHandler` in a very recent PR.

Comment (darcywong00) [reply ios/engine/KMEI/KeymanEngine/Classes/Resource Management/PackageInstallViewController.swift:352]: I see it was accidentally left in from #4338 
https://github.com/keymanapp/keyman/pull/4338/files#diff-ffe9df0bee7c059c5847b53a7f442ab3d4478d04b1d036139df0638a02949bbaR352

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by jahorton** [comment]
Found the site: https://github.com/keymanapp/keyman/pull/4338/commits/2c6096996b223f7ada938715b37205bff2af80c2

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.231-beta](https://keyman.com/downloads/releases/beta/14.0.231)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.15-alpha](https://keyman.com/downloads/releases/alpha/15.0.15)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.16-alpha](https://keyman.com/downloads/releases/alpha/15.0.16)

**Pull request merged**